src/app2.js

```diff
@@ -16,19 +17,26 @@ class App extends React.Component {
       <Router>
         <div>
           <nav className="nav">
-            <h1>Bookstore CMS</h1>
+            <h1 className="nav-header">Bookstore CMS</h1>
             <ul className="nav-item">
               <li>
-                <Link to="/books">BOOKS</Link>
+                <Link to="/">BOOKS</Link>
               </li>
               <li>
                 <Link to="/categories">CATEGORIES</Link>
               </li>
             </ul>
+            <img
+              src={user}
+              alt="User Icon"
+              width="50"
+              height="50"
+              className="nav-image"
+            />
           </nav>
         </div>
         <Routes>
-          <Route path="/books" element={<Books />} />
+          <Route exact path="/" element={<Books />} />
```

Comment:
[OPTIONAL] I found a couple of bugs in the behavior of the App. It would be nice if you could try to correct them

1. App fetches for books again after changing to another component and coming back to `Books`. Can you please think of a way to stop this from happening?

![bookstore-rerender](https://user-images.githubusercontent.com/79114233/146649344-8b5a8349-d930-4294-b200-ce5da5bfc89a.gif)

2. When a book is removed, the list does not update correctly. This leaves us with an empty list afterward. It would be good to fix this aswell.
![no-update-delete](https://user-images.githubusercontent.com/79114233/146649379-335a72ba-df09-4426-9214-acf18a8afad1.gif)